big_tests/tests/connect_SUITE.erl

```diff
@@ -709,7 +755,8 @@ configure_c2s_listener(Config, ExtraC2SOpts, RemovedC2SKeys) ->
     mongoose_helper:restart_listener(mim(), NewC2SListener).
 
 tls_opts(Mode, Config) ->
-    ExtraOpts = #{mode => Mode, cacertfile => ?CACERT_FILE, certfile => ?CERT_FILE, dhfile => ?DH_FILE},
+    %% we set `disconnect_on_failure` to false, because we do not provide 
+    ExtraOpts = #{mode => Mode, cacertfile => ?CACERT_FILE, certfile => ?CERT_FILE, dhfile => ?DH_FILE, verify_mode => none},
```

Comment:
We try to limit line length to 100 characters. It's not a must, but here I think we could easily break it in two.

Reply:
fixed